core-services/release-controller/_releases/release-ocp-4.10.json

```diff
@@ -26,6 +26,7 @@
   "verify":{
     "aws":{
       "maxRetries": 3,
+      "optional":true,
```

Comment:
this is now enforced via periodic-ci-openshift-release-master-nightly-4.10-e2e-aws-upgrade